packages/mantine/src/components/date-range-picker/DateRangePickerInlineCalendar.tsx

```diff
@@ -37,7 +37,7 @@ export interface DateRangePickerInlineCalendarProps
     /**
      * Props for RangeCalendar displayed in the popover
      */
-    rangeCalendarProps?: Omit<
+    rangeCalendarProps?: {maxDate?: Date} & Omit<
         CalendarBaseProps,
         'value' | 'onChange' | 'isDateInRange' | 'isDateFirstInRange' | 'isDateLastInRange'
     >;
```

Comment:
Can we also use minDate? I'm thinking just adding one prop here will be a temporary fix, maybe `CalendarBaseProps ` is not the correct interface anymore and we should use `DatePickerBaseProps` instead?

Reply:
This type is becoming quite an eyesore. 
If we can find something else than `{customKey: type} & omit<interface, 'a' | 'bunch' | 'of | 'stuff'>` that would be nice. How many properties there is ?